tests/fsapt-d/CMakeLists.txt

```diff
@@ -1,3 +1,3 @@
 include(TestingMacros)
 
-add_regression_test(sapt1 "psi;quicktests;smoketests;sapt;cart")
+add_regression_test(fsaptd "psi;quicktests;smoketests;sapt;cart")
```

Comment:
I'm pretty sure this label needs to match the directory name exactly. Does the test run with `ctest`? May need to add a blank line to `tests/CMakeLists.txt` to retrigger collection.